integration-tests/swiss-knife/src/main/scala/README.md

```diff
@@ -0,0 +1 @@
+# This is an example project that illustrates the use of the Spark Support in Cloudflow
```

Comment:
This file can be removed since it is just a leftover from a template. It also contains incorrect data.